backend/entities/section_application_table.py

```diff
@@ -0,0 +1,19 @@
+"""Join table of membership between Application and Section entities.""" ""
+
+from sqlalchemy import Table, Column, ForeignKey, Integer
+from .entity_base import EntityBase
+
+
+__authors__ = ["Abdulaziz Al-Shayef, Ben Goulet"]
+__copyright__ = "Copyright 2023"
+__license__ = "MIT"
+
+# Define the section_application table to be used as a join table to persist a
+# many-to-many relationship between the section and application table.
+section_application_table = Table(
```

Comment:
I think are moving towards a new convention where association tables are modeled as entities rather than as `Table` objects, so it might be good to convert this to an entity - will ping @KrisJordan to confirm this

Reply:
Yes! This is part of the refactor that we plan on doing after launch!